taxonomies/food/ingredients.txt

```diff
@@ -22956,19 +22950,36 @@ es: mezcla de cereales y semillas, mix de cereales y semillas
 hr: mješavina žitarica i sjemenki
 it: miscela di cereali e semi
 
-
-< en:cereal
-en: hydrolysed cereal, hydrolysed cereals
-fr: céréales hydrolysées
-hr: hidrolizirane žitarice
-it: idrolizzato
-nova:en: 4
-
 #<en:compound
 en: cereal with cocoa
 hr: žitarica s kakaom
 it: cereali con cacao
 
+# <en:compound
```

Comment:
So there's no more parent for the breakfast cereals? Shouldn't it be the category en:breakfast cereals ?